{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/admin.py

```diff
@@ -52,14 +53,18 @@ class CustomUserAdmin(UserAdmin):
         "is_staff",
         "is_superuser",
     )
-
+    filter_horizontal = ("groups",)
     ordering = []
 
+    def permissions(self, obj):
+        return ", ".join([g.name for g in obj.groups.all()])
+
     class Media(AutocompleteAdminMedia):
         pass
 
 
 class CustomGroupAdmin(GroupAdmin):
+    form = GroupAdminForm
```

Comment:
This has been hand added to a couple of projects now. Bringing it in here.
It improves the admin so you can immediately create permission groups and assign them to users.